### ISSUE TITLE
find a proper maintainer and hand-over to prometheus-community

### ISSUE THREAD

**Issue by lucab** [lucab/local_exporter#16]
I've sketched the PoC for this exporter to scratch my own itches (see https://github.com/lucab/local_exporter/issues/15#issuecomment-512209079), but at this point I don't have cycles to maintain this in the longer term.

Thus, this is officially looking for a proper maintainer and a new home (i.e. outside of my GH account).

From out-of-band chats:
 * @pgier and @simonpasquier were looking into the technical details, as it may fit into some other usecases of their
 * @metalmatze pointed out https://github.com/prometheus-community/community
 * if somebody steps up to maintain it, it could be moved to the `prometheus-community` orga

**Comment by simonpasquier** [comment]
Agreed, prometheus-community would be the natural place where to move it.

**Comment by anthr76** [comment]
Has any traction been made on this? I'd at the very least like to bring it up to date with libraries and container image to avoid some possible vulnerabilities
